LL/n_echach.trig

```diff
@@ -1440,9 +1448,9 @@
     owl:sameAs <#Cormac-7bffbd20>.
 
 # While a Dubchormac does appear above (<#Aires>), it hardly seems
-# likely that he would appear here. My theory is that <#AedDub>, a son 
-# of <#Cormac-7bffbd20>, was originally written but that this has been 
-# mangled in copying. But can we reconstruct that or should we just 
+# likely that he would appear here. My theory is that <#AedDub>, a son
+# of <#Cormac-7bffbd20>, was originally written but that this has been
+# mangled in copying. But can we reconstruct that or should we just
 # include what was written? EPT
```

Comment:
Should we remove this comment?  I am wondering that because the policy is that we put down what is written unless our electronic sources are incorrect in some way.

Reply:
The comment is all about whether to `<#Duibcormaic>` is the same as `<#Cormac-7bffbd20>`, as the latter is more plausible, given the genealogical situation. Determining sameAs is the one area where we are often not encoding what has been written explicitly in the text.